src/main/scala/nl.knaw.dans.easy.managedeposit/EasyManageDepositApp.scala

```diff
@@ -319,7 +332,8 @@ class EasyManageDepositApp(configuration: Configuration) extends DebugEnhancedLo
       case Deposit(_, _, _, "FAILED", _, _, _, _, _) => true
       case Deposit(_, _, _, "REJECTED", _, _, _, _, _) => true
       case Deposit(_, _, _, null, _, _, _, _, _) => true
-      // TODO add case for state == ARCHIVED && doiRegistered == false
+      // When the doi of an archived deposit is registered an error should be raised
+      case Deposit(_, Identifier(_, Doi(_, Some(true))), _, "ARCHIVED", _, _, _, _, _)  => true
```

Comment:
I think it should raise an error when the archived deposit is NOT registered...

```suggestion
      case Deposit(_, Identifier(_, Doi(_, Some(false))), _, "ARCHIVED", _, _, _, _, _)  => true
```